src/server/graphql/models/Meeting/meetingSubscription.js

```diff
@@ -1,38 +1,44 @@
-import r from '../../../database/rethinkdriver';
-import {isLoggedIn} from '../authorization';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
+// import {isLoggedIn} from '../authorization';
 import {getFields} from '../utils';
 import {Meeting} from './meetingSchema';
-import {GraphQLNonNull, GraphQLID} from 'graphql'
+import {GraphQLNonNull, GraphQLID} from 'graphql';
 
 export default {
   getMeeting: {
     type: Meeting,
     args: {
-      meetingId: {type: new GraphQLNonNull(GraphQLID), description: 'The unique meeting ID'}
+      meetingId: {
+        type: new GraphQLNonNull(GraphQLID),
+        description: 'The unique meeting ID'
+      }
     },
     async resolve(source, {meetingId}, refs) {
       const {rootValue} = refs;
+      // eslint-disable-next-line no-unused-vars
```

Comment:
for all these i say we let it ride on this PR since the 0.5.0 PR will change them all anyways